turbinia/workers/__init__.py

```diff
@@ -951,95 +951,93 @@ def run_wrapper(self, evidence):
             message=message, trace=traceback.format_exc())
       return self.result.serialize()
 
-    with filelock.FileLock(config.LOCK_FILE):
-      log.info('Starting Task {0:s} {1:s}'.format(self.name, self.id))
-      original_result_id = None
-      turbinia_worker_tasks_started_total.inc()
-      task_runtime_metrics = self.get_metrics()
-      with task_runtime_metrics.time():
-        try:
-          original_result_id = self.result.id
-
-          # Check if Task's job is available for the worker.
-          active_jobs = list(job_manager.JobsManager.GetJobNames())
-          if self.job_name.lower() not in active_jobs:
-            message = (
-                'Task will not run due to the job: {0:s} being disabled '
-                'on the worker.'.format(self.job_name))
-            self.result.log(message, level=logging.ERROR)
-            self.result.status = message
-            return self.result.serialize()
-
-          self.evidence_setup(evidence)
-
-          if self.turbinia_version != turbinia.__version__:
-            message = (
-                'Worker and Server versions do not match: {0:s} != {1:s}'
-                .format(self.turbinia_version, turbinia.__version__))
-            self.result.log(message, level=logging.ERROR)
-            self.result.status = message
-            self.result.successful = False
-            return self.result.serialize()
-
-          self.result.update_task_status(self, 'running')
-          self._evidence_config = evidence.config
-          self.task_config = self.get_task_recipe(evidence.config)
-
-          self.result = self.run(evidence, self.result)
-
-        # pylint: disable=broad-except
-        except Exception as exception:
+    log.info('Starting Task {0:s} {1:s}'.format(self.name, self.id))
```

Comment:
Just to confirm so I don't need to check all this -- I'm assuming the only change here is the `with` statement being removed and everything else is just dedented?

Reply:
Yep just removed the with statement and dedented